wgsl/index.bs

```diff
@@ -694,6 +689,39 @@ or a [=hexadecimal floating point literal=].
   </xmp>
 </div>
 
+<div algorithm="mathematical value of decimal floating point literal">
+The mathematical value of a [=decimal floating point literal=] is computed as follows:
+* Compute |effective_mantissa| from |mantissa|:
+    * If |mantissa| has 20 or fewer [=mantissa/significant=] digits, then |effective_mantissa| is |mantissa|.
+    * Otherwise:
+        * Let |truncated_mantissa| be the same as |mantissa| except each digit to the
+             right of the 20th significant digit is replaced with 0.
```

Comment:
Would it make sense to just make the 20 a lower bound? So, the implementation has to handle at least 20, but can do what it wants after that?

Reply:
Hm.  The committee decision was basically "let's do what ECMAScript does", and I didn't think about it too much.

WGSL is different in that AbstractFloat is IEEE float64 right now, but could be expanded to more later.  So in that sense it's different from ECMAScript, which is fixed at IEEE float64.

Actually thinking about it again:
* IEEE binary64 has 53 bits mantissa (including hidden 1 for normal numbers), and that's less than the ~66.4 bits captured by 20 significant digits.  
* So is it true that it kind of doesn't matter that it's 20, as long as it has at least enough significant digits to fill 53 bits of mantissa. 
* Which means we could radically simplify?  (But need to take it back to the group for review)

Reply:
I traced the "20 digits" rule back to the initial commit in https://github.com/tc39/ecma262 in 2014-04-09

And it also appears in the June 1997 edition of ECMAscript: https://www.ecma-international.org/wp-content/uploads/ECMA-262_1st_edition_june_1997.pdf